src/commands/tileindex-validate/__test__/tileindex.validate.test.ts

```diff
@@ -276,3 +278,14 @@ describe('GridSizeFromString', () => {
     );
   });
 });
+
+describe('is8BitsTiff', () => {
+  it('should be a 8bits Tiff', async () => {
```

Comment:
- "TIFF" is an abbreviation, so it should be all upper case
- "8-bit" or "8 bit" are the normal ways to format that part of the string.